src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -133,12 +141,12 @@ public void parse_allFieldsSpecified_success() {
         Index targetIndex = INDEX_SECOND_PERSON;
         String userInput = targetIndex.getOneBased() + IC_DESC_AMY + PHONE_DESC_BOB + TAG_DESC_HUSBAND
                 + EMAIL_DESC_AMY + ADDRESS_DESC_AMY + HEIGHT_DESC_AMY + WEIGHT_DESC_AMY + NAME_DESC_AMY
-                + TAG_DESC_FRIEND;
+                + TAG_DESC_FRIEND + DOB_DESC_AMY;
```

Comment:
Maybe you can re-arrange the order of the fields to make it more consistent with the rest of the code

Reply:
Fixed